src/object.d

```diff
@@ -338,7 +337,10 @@ class TypeInfo_Typedef : TypeInfo
 
     override @property inout(TypeInfo) next() nothrow pure inout { return base.next; }
     override @property uint flags() nothrow pure const { return base.flags; }
-    override const(void)[] init() nothrow pure const @safe { return m_init.length ? m_init : base.init(); }
+    override const(void)[] init() nothrow pure const @trusted @nogc
+    { 
+        return m_init.length ? m_init : (cast(void *)null)[0..0]; 
```

Comment:
You are changing behavior here, I don't think you need to make any modifications.